src/Knp/Rad/ResourceResolver/Parser/ServiceBased/ImplicitConfigParser.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Knp\Rad\ResourceResolver\Parser\ServiceBased;
+
+use Knp\Rad\ResourceResolver\Caster;
+use Knp\Rad\ResourceResolver\Parser;
+use Knp\Rad\ResourceResolver\Resource\CallableResource;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+final class ImplicitConfigParser implements Parser
+{
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    /**
+     * @var Caster\Container
+     */
+    private $casters;
+
+    /**
+     * @param ContainerInterface $container
+     * @param Caster\Container   $casters
+     */
+    public function __construct(ContainerInterface $container, Caster\Container $casters)
+    {
+        $this->container = $container;
+        $this->casters   = $casters;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function parse($name, $config)
+    {
+        list($serviceName, $methodName) = explode(':', $config[0]);
```

Comment:
So the notation is `service_name:method`. Why not use `@service_name:method` so we can also implements the following syntax `Namespace\Classname:method` ?